docs/overview/Glossary.md

```diff
@@ -337,6 +337,10 @@ A write-in is a [candidate or option](#candidate) which does not appear on the [
 
 A zero-knowledge proof is a method by which one party (the prover) can prove to another party (the verifier) that they know a value `x`, without conveying any information apart from the fact that they know the value `x`. The essence of zero-knowledge proofs is that it is trivial to prove that one possesses knowledge of certain information by simply revealing it; the challenge is to prove such possession without revealing the information itself or any additional information.[^2]
 
+### sequence order
```

Comment:
Great. One small change to move this up as these are in alphabetical order.

Reply:
ok great will do that now

Reply:
Saw the updates. See my comments and this is good to go after that.